sync/sync.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd ) ; readonly dir
+cd "${dir}/.."
+
+set -x
+# Sync using vendir
+vendir sync
+{ set +x; } 2>/dev/null
+
+# Patches
+./sync/patches/crds/patch.sh
+./sync/patches/values-schema/patch.sh
+./sync/patches/affinity-topologyspreadconstraints/patch.sh
+./sync/patches/psp/patch.sh
+./sync/patches/servicemonitor/patch.sh
+./sync/patches/gs-helpers/patch.sh
+./sync/patches/vpa/patch.sh
+./sync/patches/helmignore/patch.sh
+./sync/patches/kube-linter/patch.sh
+
+# Store diffs
+rm -f ./diffs/*
+for f in $(git --no-pager diff --no-exit-code --no-color --no-index vendor/kong/charts/kong helm/kong-app --name-only) ; do
+        set +e
```

Comment:
since we are ignoring `Chart.yaml` and `Chart.json` in vendir, I think we should also ignore it here; otherwise CI will complain, right?

Reply:
CI did not complain in https://github.com/giantswarm/kong-app/pull/401/files#diff-1a7cd77dea53cb9301d55666b5b86c4661cc87a54cabae4f4a0f8afa34d9f6aa

Reply:
Sorry I meant in the release PR

Reply:
There is another problem: Since the `Create Release PR` action is pushing using a default Github actions token, our CI check is not triggered.

But after pushing a commit manually, it seems you're right! https://github.com/giantswarm/kong-app/actions/runs/10038801133/job/27741478437?pr=403

See https://github.com/giantswarm/kong-app/pull/403

Reply:
It works with this change https://github.com/giantswarm/kong-app/pull/399/commits/335ddd38bb01051426b981874c3d4ceb8c527015

See https://github.com/giantswarm/kong-app/pull/405